client/src/component/CohortsTable.js

```diff
@@ -0,0 +1,20 @@
+import { Button } from "./Button";
+
+export const CohortsTable = () => {
+	return (
+		<table>
+			<thead>
+				<tr>
+					<th>name</th>
+					<th>view</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>LDN9</td>
+					<td> <Button title={"view"} /></td>
+				</tr>
+			</tbody>
+		</table>
+	);
+};
```

Comment:
yes maybe if it is in separate pull request will be better but on the tickets all were together because I want to reduce the number of pull request and go forward